src/Persistence/Initialization/VersionSeasonSix/ChaosMixes.cs

```diff
@@ -373,10 +368,9 @@ private ItemCrafting ThirdWingsStage1Crafting()
         crafting.Name = "3rd Level Wings, Stage 1";
         crafting.Number = 38;
         var craftingSettings = this.Context.CreateNew<SimpleCraftingSettings>();
-
         crafting.SimpleCraftingSettings = craftingSettings;
-        craftingSettings.MoneyPerFinalSuccessPercentage = 200000;
-        craftingSettings.SuccessPercent = 0;
+        craftingSettings.MoneyPerFinalSuccessPercentage = 200_000;
+        craftingSettings.SuccessPercent = 1;
```

Comment:
https://github.com/kyleruss/emu-server/blob/202856a74c905c203b9b2795fd161f564ca8b257/GameServer/Source/ChaosBox.cpp#L4635
According to my tests, the client always adds 1% to the calculated rate.